### PR TITLE
refactor filters to TS

### PR DESCRIPTION
**Pull request by ratroian**

### CODE REVIEW

File: src/scripts/filters/filters.ts [hunk 1]
```diff
@@ -4,12 +4,12 @@ import { toggleClassFilters } from '../movies/filters';
...
 import { disableLoadMoreBtn } from '../movies/render-movies';
 
-const form = document.querySelector('#filters');
-const disabledButton = document.querySelector('#reset-btn');
-const filterButton = document.querySelector('#filter-btn');
-const formItems = form.querySelectorAll('[data-url]');
+const form: HTMLElement = document.querySelector('#filters');
+const disabledButton: HTMLButtonElement = document.querySelector('#reset-btn');
+const filterButton: HTMLButtonElement = document.querySelector('#filter-btn');
+const formItems: NodeListOf<HTMLFormElement> = form.querySelectorAll('[data-url]');
```
Comment (duhenets) [inline src/scripts/filters/filters.ts:10]: HTMLInputElement for input element

File: src/scripts/filters/filters.ts [hunk 2]
```diff
@@ -4,12 +4,12 @@ import { toggleClassFilters } from '../movies/filters';
 import { getNextPage } from '../movies/requests';
 import { disableLoadMoreBtn } from '../movies/render-movies';
 
-const form = document.querySelector('#filters');
-const disabledButton = document.querySelector('#reset-btn');
-const filterButton = document.querySelector('#filter-btn');
-const formItems = form.querySelectorAll('[data-url]');
+const form: HTMLElement = document.querySelector('#filters');
```
Comment (duhenets) [inline src/scripts/filters/filters.ts]: HtmlFormElement

File: src/scripts/movies/FilterRange.ts [hunk 3]
```diff
@@ -1,5 +1,23 @@
...
+export default class FilterRange implements TFilterRangeConstructor {
+    parentRange: Element;
+
+    infoMin: HTMLInputElement;
+
+    infoMax: HTMLInputElement;
+
+    rangeMin: HTMLInputElement;
+
+    rangeMax: HTMLInputElement;
```
Comment (duhenets) [inline src/scripts/movies/FilterRange.ts]: if inside the class, then a separate type is not needed

File: src/styles/films.css [hunk 4]
```diff
@@ -47,7 +49,18 @@
     grid-gap: 20px;
     justify-content: center;
     flex-grow: 1;
+    /*display: flex;*/
+    /*justify-content: center;*/
+    /*align-items: center;*/
```
Comment (duhenets) [inline src/styles/films.css]: delete comments

File: src/styles/films.css [hunk 5]
```diff
@@ -150,3 +163,21 @@
...
+.not-fond-wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    top: 40%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    /*display: none;*/
```
Comment (duhenets) [inline src/styles/films.css]: delete comments

**Review by duhenets — CHANGES_REQUESTED** [review]
fix comments

File: src/scripts/filters/filters.ts (hunk 1 above)
Comment (ratroian) [reply src/scripts/filters/filters.ts:10]: Done. HTMLFormElement

File: src/scripts/filters/filters.ts (hunk 2 above)
Comment (ratroian) [reply src/scripts/filters/filters.ts]: Done

File: src/scripts/movies/FilterRange.ts (hunk 3 above)
Comment (ratroian) [reply src/scripts/movies/FilterRange.ts]: Ok

File: src/styles/films.css (hunk 4 above)
Comment (ratroian) [reply src/styles/films.css]: Done

File: src/styles/films.css (hunk 5 above)
Comment (ratroian) [reply src/styles/films.css]: Done

**Review by duhenets — APPROVED** [review]

**Pull request merged**